doc/build/dialects/postgresql.rst

```diff
@@ -154,6 +154,68 @@ For example:
       during=DateTimeRange(datetime(2013, 3, 23), None)
   )
 
+MultiRange Types
+~~~~~~~~~~~
```

Comment:
**mike bayer** (zzzeek) wrote:

extend underline for rst

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3728